src/python/pyhbac.c

```diff
@@ -1932,7 +1932,6 @@ initpyhbac(void)
 #endif
 {
     PyObject *m;
-    int ret;
```

Comment:
Is there strong reason to get rid of `ret`?
Frankly I would leave it intact just for the sake of reduced diff...